rules.d/70-memory.rules

```diff
@@ -0,0 +1,8 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="memory_end"
```

Comment:
Hmm, "remove|unbind" ?

Reply:
"unbind" shouldn't happen for DMI devices. And even if, I think the rules should run too and update the data given what is available then. i.e. if the "unbind" means some attrs vanish, then great, the rules will try to read them, notice they don#t exist anymore, and handle that gracefully and not generate props.

i'd always write stuff gracefully. Unless we know for sure the whole thing is gone (i.e. on "remove") let's run as much as we can, but make sure to write the rules reasonably graceful.